aws/eks/external_secrets.tf

```diff
@@ -0,0 +1,40 @@
+data "aws_iam_policy_document" "external_secrets" {
+  statement {
+    actions = [
+      "secretsmanager:GetSecretValue",
+    ]
+
+    resources = [
+      "arn:aws:secretsmanager:us-west-2:${data.aws_caller_identity.current.account_id}:secret:${var.secrets_path}",
```

Comment:
@limed What would be the best way of getting secrets_path in here?

Reply:
I think for now this is good, but maybe configure `${var.secrets_path}` as a list and create a loop through the paths for the variables.

Reply:
thanks this makes sense